sdk/storage/tests.datamovement.yml

```diff
@@ -15,7 +15,7 @@ extends:
     SupportedClouds: Public #, PrivatePreview
     MatrixReplace:
       # Use dedicated storage pool in canadacentral with higher memory capacity
-      - Pool=.*LINUXPOOL.*/azsdk-pool-mms-ubuntu-2004-storage
```

Comment:
@benbp @mikeharder do you know if we should also update the storage pools?

Reply:
Yes, let's do it.  Added you to mail thread with Storage.